/*
 * rational.cpp

 *
 *  Created on: Oct 20, 2015
 *      Author: rtakacsux
 */
#include <stdio.h>
#include "rational.h"

int main(){

	try{
		rational r;
		std::cout << r.numerator() << "/" << r.denominator() << '\n';
		r = 2;
		std::cout << r.numerator() << "/" << r.denominator() << '\n';

	}catch( rational::exception const &r){
		std::cerr << "Exception from rational caught.\n";
	}
}



